tools/PrebidValidator/Dr.Prebid/TestSummary/AdServerValidation/PBVLineItemsSetupValidator.m

```diff
@@ -89,13 +89,13 @@ - (void)startTest
     NSString *adUnitID = [[NSUserDefaults standardUserDefaults] stringForKey:kAdUnitIdKey];
     NSString *bidPrice = [[NSUserDefaults standardUserDefaults] stringForKey:kBidPriceKey];
         
-    GADAdSize GADAdSize = kGADAdSizeInvalid;
+    GADAdSize GADAdSize = GADAdSizeInvalid;
```

Comment:
Clarify why we have the changes in the file

Reply:
This is because it has been changed by GAD and the name has changed now.